asf-site-src/source/contribute/releases.html.md

```diff
@@ -552,37 +583,16 @@ Once a release candidate obtains majority approval from the PMC, there are sever
         Meg Smith
         Apache Yetus PMC
     If you'd like feedback on the draft, feel free to post it for review on your release issue.
-1. Wait 24 hours for mirrors to get properly updated.
+1. Wait 24 hours for mirrors to get properly updated before continuing.
 
 ### Documentation
 
-1. Update the documentation in the git main branch for the new release.  Remove the oldest release and add the latest.
-
-   ```bash
-   $ release/update-doc-versions.sh --version=<x.y.z -- version WITHOUT the rel/!>
-   $ git add -p
-   $ git add asf-site-src/pom.xml
-   $ git commit
-   ```
-
-   - Example commit message:
-
-   ```text
-   YETUS-XXX. add release 0.7.0.
-
-   - list in releases
-   - remove 0.4.0, add 0.7.0 to pom.xml
-   ```
-
-1. You should then post this patch for review. Once you've gotten feedback, it's fine to push the patch to the ASF source repo immediately so long as the updated website is not published.
-1. 24 hours after the distribution repo has been updated, publish website updates. See [Maintaining the Yetus Website](../website).
+1. Publish website updates. See [Maintaining the Yetus Website](../website).
 1. Verify that https://yetus.apache.org/latest.tgz and https://yetus.apache.org/latest.tgz.asc download the newly released version.
 
 ### Homebrew
 
-24 Hours after the distribution repo has been updated, update Homebrew to point to the new release:
-
-1. Update the `yetus-homebrew` repo by using the release script:
+1. Update the `yetus-homebrew` repo by using the release script. It will tag _and sign_ (so GPG needs to work!) the top of the tree to point to the release. (Homebrew only uses top of tree, so branches are pointless.)
```

Comment:
Is there a way to verify that GPG is working before taking destructive actions?

Reply:
The sign happens as part of the commit so one can re-run the release script as many times as necessary if GPG is broken.  But agree that it would be useful to have a 'verify GPG is working for me'-type check.  The build-and-sign script for the core yetus repo has _some_ checks but could be beefed up.  Also, the checks for homebrew, etc, happen a week+ after build-and-sign was executed so it may not work any more.

I'll open a new JIRA issue to take in that improvement.

Thanks!